api/src/main/java/au/org/aodn/nrmn/restapi/repository/SpeciesWithAttributesRepository.java

```diff
@@ -0,0 +1,27 @@
+package au.org.aodn.nrmn.restapi.repository;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.stereotype.Repository;
+
+import au.org.aodn.nrmn.restapi.model.db.SpeciesWithAttributes;
+import au.org.aodn.nrmn.restapi.repository.projections.SpeciesWithAttributesCsvRow;
+
+@Repository
+public interface SpeciesWithAttributesRepository
+        extends JpaRepository<SpeciesWithAttributes, Integer>, JpaSpecificationExecutor<SpeciesWithAttributes> {
```

Comment:
do you need this ? this repo was removed longtime ago

Reply:
It's still used in getSpeciesForTemplate.

Reply:
ui_species_attribute is pulled from ObservationRepository as a query 
https://github.com/aodn/nrmn-application/blob/master/api/src/main/java/au/org/aodn/nrmn/restapi/repository/ObservationRepository.java